source/manual/mysql.html.md

```diff
@@ -5,49 +5,55 @@ section: Backups
 type: learn
 layout: manual_layout
 parent: "/manual.html"
-last_reviewed_on: 2019-08-02
+last_reviewed_on: 2020-02-05
 review_in: 6 months
 ---
 
 ## automysqlbackup
 
-This is how MySQL backups have traditionally been taken on the GOV.UK Infrastructure.
+### Backing up
+We use a third-party script called [automysqlbackup](https://github.com/alphagov/govuk-puppet/blob/master/modules/govuk_mysql/templates/automysqlbackup) to take MySQL backups of GOV.UK infrastructure.
 
-A third-party script called [automysqlbackup](https://github.com/alphagov/govuk-puppet/blob/master/modules/govuk_mysql/templates/automysqlbackup) takes a `mysqldump` every night and stores them on disk
-on a dedicated mount point on the MySQL backup machines (mysql-backup-1.backend and whitehall-mysql-backup-1.backend).
+This script takes a nightly `mysqldump` and stores it on a dedicated mount point on the MySQL backup machines (`mysql-backup-1.backend` and `whitehall-mysql-backup-1.backend`).
 
-The onsite backup machine (backup-1.management) pulls the latest backup and stores it on disk. [Duplicity](http://duplicity.nongnu.org/)
-runs each night to send encrypted backups to an Amazon S3 bucket.
+The on-site backup machine (`backup-1.management`) pulls the latest backup and stores it on disk. [Duplicity](http://duplicity.nongnu.org/), runs nightly to send encrypted backups to an Amazon S3 bucket.
 
+### Restoring
 To restore from this method:
 
- - Fetch a backup from either the dedicated mount point, the onsite machine or the S3 bucket [using duplicity](restore-from-offsite-backups.html) (to decrypt you may need a password kept in encrypted hieradata).
+ - Fetch a backup from either the dedicated mount point, the on-site machine, or the S3 bucket [using duplicity](restore-from-offsite-backups.html) (to decrypt you may need a password kept in encrypted hieradata).
  - Unzip the file
- - Import into MySQL using `mysql < <file>`
+ - Import into MySQL using `mysql < file` (see these [MySQL docs](https://dev.mysql.com/doc/refman/8.0/en/mysql-batch-commands.html) on using file imports).
```

Comment:
Suggest this to make the link stand alone a bit - sometimes it could be viewed without the surrounding context. Plus, again, removing parenthesises :)

```suggestion
 - Import into MySQL using `mysql < file` - see these [MySQL docs on using file imports](https://dev.mysql.com/doc/refman/8.0/en/mysql-batch-commands.html).
```